BlueRings.py

```diff
@@ -0,0 +1,156 @@
+import scipy.ndimage as  ndimage
+import pylab as plt
+import numpy
+
+def getPeak(img,x,y,boxsize=10):
+    from scipy import ndimage
+    img = img*1
+    a=img.shape[0]/2-5
+    img[:a,:]=0
+    img[-a:,:]=0
+    img[:,:a]=0
+    img[:,-a:]=0
+   
+    for i in range(1):
+        d = ndimage.gaussian_filter(img,1.)
+        Y,X = numpy.mgrid[0:d.shape[0],0:d.shape[1]]
+        Y -= Y.mean()
+        X -= X.mean()
+        d /= d.sum()
+        x = x+int(numpy.round((d*X).sum()))
+        y = y+int(numpy.round((d*Y).sum()))
+    return x,y
+
+
+def convolve(image,psf,doPSF=True,edgeCheck=True):
+    """
```

Comment:
I'm trying to adopt the `numpy` standard for docstrings, so that if I ever run sphinx it will make something sensible. You can read about this [here](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#docstring-standard)